src/PerfView/EventViewer/EventWindow.xaml.cs

```diff
@@ -1154,18 +1154,18 @@ public void Update()
             }
             else
             {
-                var match = Regex.Match(StartTextBox.Text, @"^\s*([\d\.,]+)\s+([\d\.,]+)\s*$");
-                if (match.Success)
+                if (RangeUtilities.TryParse(StartTextBox.Text, out var startTimeRelativeMSec, out var endTimeRelativeMSec))
                 {
-                    m_source.StartTimeRelativeMSec = 0;
-                    double.TryParse(match.Groups[1].Value, out m_source.StartTimeRelativeMSec);
-
-                    m_source.EndTimeRelativeMSec = m_source.MaxEventTimeRelativeMsec;
```

Comment:
The expected behavior here is that if we can't properly parse, then the range should be [0, MaxEventTimeRelativeMsec].  `RangeUtilities.TryParse` will set both of these to 0 if it can't properly parse an individual value.

Reply:
Not exactly. We "if we can't match on the Regex". Because, after that, it's whatever `double.TryParse` outputs. Lines 1160 and 1163 are, effectively, no-op.